casepro/pods/registry.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+import importlib
+
+
+def get_class_from_string(string):
+    '''
+    Given a class specified by a string, this will return that class.
+    '''
+    module_name, cls_name = string.rsplit('.', 1)
+    module = importlib.import_module(module_name)
+    return getattr(module, cls_name)
+
+
+def load_pod(index, config):
+    '''
+    Given the index of the pod, and the config dictionary for the pod, this
+    returns the instance of that pod.
+    '''
+    class_name = config.pop('type')
+    config['index'] = index
+    return get_class_from_string(class_name)(config)
+
+
+pods = [
```

Comment:
Maybe this should be a tuple rather than a list? Appending to pods later will result in URL patterns not being set up correctly.